server/prisma/generator/factories/instanceRoles.factory.ts

```diff
@@ -1,6 +1,10 @@
 import { prisma } from '../../../src/prisma';
 
-import { InstancePermission, Permission } from '../../../../common/permissions';
+import {
+  ChapterPermission,
+  InstancePermission,
+  Permission,
+} from '../../../../common/permissions';
```

Comment:
```suggestion
import { InstancePermission, Permission } from '../../../../common/permissions';
```
We members should not be able to ban users, so we do not need to pull out the chapter permissions here.